src/app/components/challenge/challenge.component.html

```diff
@@ -125,7 +125,7 @@
                                     [routerLink]="['./leaderboard']"
                                     class="text-light-black fw-semibold fs-15">
                                     <i class="fa fa-th-list"></i> Leaderboard</a></li>
-                            <li *ngIf=""><a
+                            <li *ngIf="challenge.forumURL"><a
```

Comment:
It will be better to add condition for `enable forum` field also.

Reply:
I don't understand @Sanji515

Reply:
I get it now @Sanji515 ....Done!!